pkg/assembler/backends/neo4j/certifyVuln.go

```diff
@@ -397,3 +379,276 @@ func generateModelCertifyVuln(pkg *model.Package, vuln model.OsvCveGhsaObject, t
 	}
 	return &certifyVuln
 }
+
+//  Ingest Vulnerability
+
+func (c *neo4jClient) IngestVulnerability(ctx context.Context, pkg model.PkgInputSpec, osv *model.OSVInputSpec, cve *model.CVEInputSpec, ghsa *model.GHSAInputSpec, certifyVuln model.CertifyVulnInputSpec) (*model.CertifyVuln, error) {
+
+	err := helper.CheckIngestVulnInputs(osv, cve, ghsa)
+	if err != nil {
+		return nil, err
+	}
+
+	session := c.driver.NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
+	defer session.Close()
+
+	var sb strings.Builder
+	var firstMatch bool = true
+	queryValues := map[string]any{}
+
+	queryValues[timeScanned] = certifyVuln.TimeScanned
```

Comment:
```suggestion
	queryValues[timeScanned] = certifyVuln.TimeScanned.UTC()
```

Based on #538. We need to make sure times are in UTC to not confuse neo4j.